scripts/west_commands/ncs-provision.py

```diff
@@ -36,8 +36,9 @@ def do_add_parser(self, parser_adder):
             "-k", "--key", type=Path, action='append', dest="keys",
             help="Input .pem file with ED25519 private key"
         )
-        upload_parser.add_argument("-p", "--policy", type=str, help="Keys policy",
-                                   choices=["revokable", "lock"], default="revokable")
+        upload_parser.add_argument("-p", "--policy", type=str,
+                                   choices=["revokable", "lock", "lock-last"], default="revokable",
+                                   help="Policy applied to the given set of keys. revokable: keys can be revoked each by one. lock: all keys stay as they are. lock-last: last key is uploaded as locked, others as revokable")
```

Comment:
nit: you can split help message into lines, to avoid long lines
```
help="Policy applied to the given set of keys."
    " revokable: keys can be revoked each by one."
    " lock: all keys stay as they are."
    " lock-last: last key is uploaded as locked, others as revokable.")
```

Reply:
> means: all keys are locked?

yes